### PR TITLE
Update form styles

### PR DESCRIPTION
**Pull request by psiemens**

### CODE REVIEW

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 1]
```diff
@@ -32,53 +35,54 @@ export default class PersonForm extends React.Component {
```
Comment (Rowansdabomb) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js:35]: Move constructor to top of Component

File: dispatch/static/manager/src/js/components/PollEditor/PollForm.js [hunk 2]
```diff
@@ -124,8 +127,9 @@ export default class PollForm extends React.Component {
     return (
       <div className={'c-poll-form-container'}>
         <div className={'c-equal-width'}>
-          <form onSubmit={e => e.preventDefault()}>
-            <FormInput
```
Comment (Rowansdabomb) [inline dispatch/static/manager/src/js/components/PollEditor/PollForm.js:128]: This is cool

**Review by Rowansdabomb — COMMENTED** [review]
Looks Good. There is just one constructor that should be moved to the top of the component for consistency.

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js (hunk 1 above)
Comment (psiemens) [reply dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js:35]: nice catch 👍

**Pull request merged**